tests/data_utils.py

```diff
@@ -3,8 +3,23 @@
 
 import json
 import os
+import pathlib
+import shutil
+import sys
+from argparse import Namespace
 from typing import Optional
 
+from omegaconf import DictConfig
+from omegaconf import OmegaConf as om
+
+from scripts.data_prep.convert_dataset_hf import main as main_hf  # noqa: E402
+from scripts.data_prep.convert_dataset_json import \
+    main as main_json  # noqa: E402
+
+# Add repo root to path so we can import scripts and test it
```

Comment:
don't you need to do this before you import `scripts`? might be missing something

Reply:
there may be some magic on previous tests running this already.. moved up just in case, likely want to add something like this in conftest instead but will save that for a later date